resolver_wrapper.go

```diff
@@ -78,7 +79,16 @@ func (ccr *ccResolverWrapper) start() error {
 			Authority:            ccr.cc.authority,
 		}
 		var err error
-		ccr.resolver, err = ccr.cc.resolverBuilder.Build(ccr.cc.parsedTarget, ccr, opts)
+		// The delegating resolver is used unless:
+		//   - A custom dialer is provided via WithContextDialer dialoption.
+		//   - Proxy usage is disabled through WithNoProxy dialoption.
```

Comment:
Nit: this list should ideally say `, or` after the first bullet, since it's not clear from the text here whether these conditions are ORs or ANDs.